__tests__/app.test.js

```diff
@@ -0,0 +1,30 @@
+const request = require("supertest");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+
+// beforeEach(() => {
+//   return seed(data);
+// });
+
+describe("GET /api/topics", () => {
+  test("Status: 200", () => {
+    return request(app).get("/api/topics").expect(200);
+  });
+  test("Status: 200, returns array of 3 test objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.topics).toHaveLength(3);
+      });
```

Comment:
This is ok but it could be more adequate with the use of the matcher expect.objectContaining. Please use it from now on as well to check the structure of your responses objects